pkgs/by-name/cu/cunicu/package.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  protobuf,
+  protoc-gen-go,
+  protoc-gen-go-grpc,
+}:
+buildGoModule rec {
+  pname = "cunicu";
+  version = "0.5.53";
+
+  src = fetchFromGitHub {
+    owner = "cunicu";
+    repo = "cunicu";
+    rev = "v${version}";
+    hash = "sha256-y8JyXf+LdquhVr9G5wX1LdMF7wpj5PZG3xXGAWoypzA=";
+  };
+
+  nativeBuildInputs = [
+    protobuf
+    protoc-gen-go
+    protoc-gen-go-grpc
+  ];
+
+  CGO_ENABLED = 0;
+
+  vendorHash = "sha256-OiLVdEf6fcGHx0k0xC5sZwhnK0FiLgfdkz2zNgBbcgY=";
+
+  # These packages contain networking dependent tests which fail in the sandbox
+  excludedPackages = [
+    "pkg/config"
+    "pkg/selfupdate"
+    "pkg/tty"
+    "scripts"
+  ];
+
+  ldflags = [
+    "-X"
+    "cunicu.li/cunicu/pkg/buildinfo.Version=${version}"
+    "-X"
+    "cunicu.li/cunicu/pkg/buildinfo.BuiltBy=Nix"
```

Comment:
```suggestion
    "-X cunicu.li/cunicu/pkg/buildinfo.Version=${version}"
    "-X cunicu.li/cunicu/pkg/buildinfo.BuiltBy=Nix"
```

Reply:
This snipped has been formatted like this by `nixfmt`.

Reply:
This is not about formatting

Reply:
Oh, I missed that.

Fixed in f8e76a836715